pkg/argo/fetch_test.go

```diff
@@ -62,6 +63,19 @@ spec:
       command: [echo]
       args: ["{{inputs.parameters.message}}"]
 `
+
+	podYAML = `apiVersion: v1
+kind: Pod
+metadata:
+  name: sqlflow-pod
```

Comment:
Can we use generated names so that this YAML can be used by multiple tests in parallel?